libraries/getOrigin/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns the origin
+ */
+export function getOrigin() {
```

Comment:
is there any restrictions on library files? What happens if you want to import another file from the library file?

Reply:
We'll probably need to cover this at a later date, with some best practices or boundaries